src/index.css

```diff
@@ -2,8 +2,73 @@
 @tailwind components;
 @tailwind utilities;
 
+:root {
+  --orb: #ffffff;
+  --accent: #fed823;
+  --accent-hover: #fed950;
+  --primary: #15a1fc;
+  --secondary: #ff7bed;
+  --tertiary: #01feb6;
+  --formal-accent: #efecd6;
+
+  --error: #ff7b83;
+  --error-dark: #3e1b1e;
+
+  --primary-dark: #0d0030;
+  --primary-light: #bff5fa;
+  --primary-dimmed: #3a00d6;
+  --primary-semi-dimmed: #4b61d5;
+  --primary-background: #1a0259;
+  --primary-dark-red: #210030;
+  --secondary-dimmed: #cf6aff;
+
+  --accent-light-transparent: #fed82369;
+  --secondary-light-transparent: #ff7bed69;
+  --accent-light-active-transparent: #fed8239e;
+  --secondary-light-active-transparent: #ff7bed9e;
+
+  --accent-semi-transparent: rgba(254, 216, 35, 0.4);
+  --primary-semi-transparent: rgba(21, 161, 252, 0.5);
+  --secondary-semi-transparent: rgba(255, 123, 237, 0.5);
+  --tertiary-semi-transparent: rgba(1, 254, 182, 0.4);
+  --formal-accent-semi-transparent: rgba(239, 236, 214, 0.4);
+  --error-semi-transparent: rgba(255, 123, 131, 0.2);
+  --selection: #ff7bed35;
+  --transparent: #00000000;
+
+  --bg-grid: conic-gradient(
+    from 90deg at 0.06rem 0.06rem,
+    var(--primary-dark) 90deg,
+    var(--primary-background) 0
+  );
+}
+
+html {
+  background-color: var(--primary-dark);
+  scrollbar-color: var(--accent) transparent;
+  scrollbar-width: thin;
+  background-size: 100% auto, 100% auto, 3rem 3rem;
+  background-repeat: no-repeat, no-repeat, repeat;
+  background-position: top, top, 0;
+  background-image: url(), url('/img/background.webp'), var(--bg-grid);
```

Comment:
This part is always changing from project to project. Let's remove it

Reply:
fixed